src/functions/genfrac.js

```diff
@@ -11,8 +11,7 @@ import * as html from "../buildHTML";
 import * as mml from "../buildMathML";
 import {calculateSize} from "../units";
 
-const htmlBuilder = (group, options) => {
-    // Fractions are handled in the TeXbook on pages 444-445, rules 15(a-e).
+const adjustStyle = (group, options) => {
```

Comment:
```suggestion
const adjustStyle = (size, originalStyle) => {
```
I think this can take only `group.size` and `options.style` as arguments.

Reply:
Done.

Reply:
I've reverted to the previous arguments because it's too late to fight `flow`. If this is important to you, I'll do it tomorrow.

Reply:
Arguments are now revised as suggested.